libs/ui/checkbox/helm/src/lib/hlm-checkbox.component.ts

```diff
@@ -67,7 +67,7 @@ export class HlmCheckboxComponent {
 	public readonly checked = model<boolean | 'indeterminate'>(false);
 
 	public readonly name = input<string | null>(null);
-	public readonly disabled = input(false, { transform: booleanAttribute });
+	public readonly disabled = model(false);
```

Comment:
I don't think that `<hlm-checkbox disabled>` will work anymore since I removed the transform attribute and we can't pass a transform attribute to the model input.
If we want to avoid breaking changes, I can suggest a few things:
- keep the old input, but create an internal signal, that'll allow two things:
  - update the internal signal each time the disabled input is updated (through an effect), which will keep the same behavior regarding the disabled input
  - update the internal signal each time the form is updated through the `setDisabledState` function.

I don't know what do you think about this ? It can reduce the impact of the current changes.